src/components/flashMessage.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react';
+import Bus from '../lib/bus';
+
+const FlashMessage = () => {
+  const intialMessage = { content: '', type: '' };
+  const [message, setMessage] = useState(intialMessage);
+
+  useEffect(() => {
+    Bus.addListener('flash', ({ content, type }) => {
+      displayMessage(content, type);
+    });
+
+    const timeoutId = setTimeout(() => setMessage(intialMessage), 2000);
```

Comment:
I know that this line was in a different issue, but now I've come to realize that maybe we need to check `if (message.content)` before setting the msg? I think `useEffect` is going to run and the message is going to be reset every two seconds if we don't check.

Reply:
Yup! you're so right 
we didn't even notice this!
Kudos to you 🙌